sbt-scalajs-bundler/src/main/scala/scalajsbundler/Webpack.scala

```diff
@@ -35,59 +37,66 @@ object Webpack {
     * @return The written file
     */
   def writeConfigFile(
-    emitSourceMaps: Boolean,
-    webpackEntryPoints: Seq[(String, File)],
-    targetDir: File,
-    log: Logger
+      emitSourceMaps: Boolean,
+      webpackEntryPoints: Seq[(String, File)],
+      targetDir: File,
+      log: Logger
```

Comment:
It would really help the review (and `git blame` in the future) if those formatting fixes were squashed with the commits that broke the formatting to begin with. Any single commit should never break formatting.

Virtually all changes in the present commit are formatting fixes that should be ported back to previous commits.